server/pypi/packages/igraph/android64_x86_64_arm64.h

```diff
@@ -0,0 +1,9 @@
+#define IEEE_8087
```

Comment:
Oh, I see from the filename that it's already supposed to cover both architectures. But there should probably still be some #ifdefs to only define the relevant things on each one.

Reply:
> Also, is it really correct to define IEEE_8087 on ARM64?

Yes, this is needed for f2c to work. Don't take the name too seriously.

Reply:
I guess you took this file from https://igraph.org/c/html/latest/igraph-Installation.html, which says it's for macOS. And all the Google search results for IEEE_8087 do mostly return Apple-related things. I'd like to see some evidence that this file is also correct for Android.

Reply:
The proper way to generate these headers is to run the `arithchk` program, if you can. The reason why that's not done automatically here (and we need pre-generated files) is that you are cross-compiling.

This f2c-based code is just a fallback. f2c hasn't had a release since 1995 as far as I know. In an ideal situation, one links to an external BLAS/LAPACK/ARPACK, but we still want to keep igraph easy to compile with minimal external dependencies (partly for use cases like this one), so we do keep the f2c-based code functional and treat it as a first-class configuration (regular tests, etc.) But we don't want to touch the innards of f2c and fix things like the name of this `IEEE_8087` macro

Reply:
It may also be possible to run the arithchk program mentioned in that link on an actual Android device to get the correct values. We've dealt with a similar situation before – see packages/chaquopy-hdf5/README.md.

Reply:
We actually do have a build of OpenBLAS, and it's used by several other recipes. But I guess that doesn't include ARPACK, in which case f2c would still be required.

Reply:
Hi,

Back in the day, i was unsure to use header from the website so what i did is to run arithchk in my Samsung and in my emulator, both resulted the same also we are using this on many builds with QA passed from some months now.

(At least 3-4 emulators and at least 5 different android has run this)

Still please do all the checks you consider necessary.